cdap-master/pom.xml

```diff
@@ -377,6 +377,12 @@
           <version>${project.version}</version>
           <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
Why does cdap-master need to depend on the extension module?

Reply:
Discussed offline, it's to ensure the extension module is built before cdap-master so it can be packaged properly.